src/native/libs/System.Security.Cryptography.Native.Android/pal_jni.h

```diff
@@ -597,6 +597,7 @@ jmethodID GetOptionalMethod(JNIEnv *env, bool isStatic, jclass klass, const char
 jfieldID GetField(JNIEnv *env, bool isStatic, jclass klass, const char* name, const char* sig) ARGS_NON_NULL_ALL;
 jfieldID GetOptionalField(JNIEnv *env, bool isStatic, jclass klass, const char* name, const char* sig) ARGS_NON_NULL_ALL;
 JNIEnv* GetJNIEnv(void);
+jint AndroidCryptoNative_InitLibraryOnLoad (JavaVM *vm, void *reserved);
```

Comment:
should we add a comment or something that clarifies this is supposed to be called by embedders who link in the static lib?